website/docs/r/advisor_suppresion.html.markdown

```diff
@@ -0,0 +1,69 @@
+---
+subcategory: "Advisor"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_advisor_suppression"
+description: |-
+  Specifies a suppression for an Azure Advisor recommendation.
+---
+
+# azurerm_advisor_suppression
+
+Specifies a suppression for an Azure Advisor recommendation.
+
+## Example Usage
+
+```hcl
+provider "azurerm" {
+  features {}
+}
+
+data "azurerm_client_config" "current" {}
+
+data "azurerm_advisor_recommendations" "example" {}
+
+resource "azurerm_advisor_suppression" "example" {
+  name              = "HardcodedSuppressionName"
+  recommendation_id = data.azurerm_advisor_recommendations.test.recommendations[0].recommendation_name
+  resource_id       = "/subscriptions/${data.azurerm_client_config.current.subscription_id}"
+  ttl               = "01:00:00:00"
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The Name which should be used for this Advisor suppression. Changing this forces a new Advisor suppression to be created.
+
+* `recommendation_id` - (Required) The ID of the Advisor recommendation to suppress. Changing this forces a new Advisor suppression to be created.
+
+* `resource_id` - (Required) The ID of the Resource to suppress the Advisor recommendation for. Changing this forces a new Advisor suppression to be created.
+
+---
+
+* `ttl` - (Optional) A optional time to live value. If omitted, the suppression will not expire. Changing this forces a new Advisor suppression to be created.
```

Comment:
what units is this in? seconds? if so should this be
```suggestion
* `ttl_in_seconds` - (Optional) A optional time to live value. If omitted, the suppression will not expire. Changing this forces a new Advisor suppression to be created.
```

Reply:
The API of Azure expects this format: `HH:MM:SS` or `DD:HH:MM:SS`.

Should I offer a go duration format for end users and the resource take care of the format that the API expects? Or is it fine the document, that this is the expected format?

Reply:
```suggestion
* `ttl` - (Optional) A optional time to live value in the format `HH:MM:SS` or `DD:HH:MM:SS`. If omitted, the suppression will not expire. Changing this forces a new Advisor suppression to be created.
```